README.md

```diff
@@ -52,9 +52,12 @@ If you have any questions please either [open an issue](https://github.com/m88i/
 ## Pre Requisites
 
 - [`kubectl` installed](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- Kubernetes or OpenShift cluster available (minishift, minikube or crc also supported)
+- Kubernetes (1.12+) or OpenShift (4.5+) cluster available (minikube or crc also supported)
```

Comment:
The CRD v1 are introduced in 1.16. So, you might want to change that.

Reply:
Thanks!